tests/test_vision.py

```diff
@@ -114,3 +114,8 @@ def test_pack_resolution() -> None:
 def test_unpack_resolution() -> None:
 	assert unpack_resolution('0x0') == (0, 0)
 	assert unpack_resolution('2x2') == (2, 2)
+
+
+def test_match_frame_color() -> None:
```

Comment:
This does not test if the colors have been actual equalized - just if the output has a shape

1. create a gray version of `target-240p.jpg` in the test setup
2. test the actual color before and after